apps/ledger-live-mobile/src/components/RootNavigator/types/BaseNavigator.ts

```diff
@@ -166,17 +166,7 @@ export type BaseNavigatorStackParamList = {
     transaction?: Transaction;
     justScanned?: boolean;
   };
-  [ScreenName.BleDevicePairingFlow]: {
-    filterByDeviceModelId?: DeviceModelId;
-    areKnownDevicesDisplayed?: boolean;
-    areKnownDevicesPairable?: boolean;
-    onSuccessAddToKnownDevices?: boolean;
-    onSuccessNavigateToConfig: {
-      navigateInput: NavigateInput;
-      pathToDeviceParam: PathToDeviceParam;
-      navigationType?: NavigationType;
-    };
-  };
+  [ScreenName.BleDevicePairingFlow]: Record<string, never>;
```

Comment:
And just: why do we need a `Record<string, never>` for the route params ? If there are no params, i think it would be better to put `undefined`

Reply:
I agree, but it threw a TS error with react navigation typings in `screens/BleDevicePairingFlow/index.tsx` that we weren't able to solve. 
Could we check deeper on it @aussedatlo ?